x-pack/plugins/observability_solution/apm/public/components/routing/templates/apm_main_template.tsx

```diff
@@ -69,6 +71,7 @@ export function ApmMainTemplate({
   const { kibanaVersion, isCloudEnv, isServerlessEnv } = kibanaEnvironment;
   const basePath = http?.basePath.get();
   const { config } = useApmPluginContext();
+  const { isEntityManagerEnabled } = useEntityManagerEnablementContext();
```

Comment:
Don't we need to check for the FF here?

Reply:
Is there a case when the "new" view is visible and the `isEntityManagerEnabled` is `false`? I mean if we have the FF flag set to `true` but `isEntityManagerEnabled` is `false` we still show the "old" view so in this case we don't want to show the new link (only when the user sees the new view for all service pages). Am I missing something?

Reply:
I think you're right. 

We might need to revisit this once we tackle this bug: https://github.com/elastic/observability-dev/issues/3736